app/mixins/event-wizard.js

```diff
@@ -48,28 +48,31 @@ export default Mixin.create(MutableArray, CustomFormMixin, {
   async saveEventData(propsToSave = []) {
     const event = this.get('model.event');
     const data = {};
-    for (const property of propsToSave) {
+    const promises = await Promise.all(propsToSave.map(property => {
```

Comment:
Use Promise.allSettled

Reply:
The last commit didn't pass. I replaced Promise.all with Promise.allSettled. What else should I do?

Reply:
See what allSettled returns